RedHatInsights/insights-advisor-frontend
# feat(RHINENG-5841): disable remediation for edge hosts in system advi…

…sor page

# Description

Associated Jira ticket: # [(RHINENG-5841)](https://issues.redhat.com/browse/RHINENG-5841)

With this Remediation button in the advisor tab on the system detail page in the inventory application will be removed when an edge system is being looked at. As edge systems can not be remediated.


# How to test the PR

1. Run this PR with the inventory app
2. Go to inventory landing page and open Immutable tab
3. Click on any immutable tab to go to the detail page
4. Open advisor tab
5. Observe that there is not Remediate button in the table toolbar
6. Navigate back to inventory landing page, conventional tab
7. Click on any conventional system to go to the detail page
8. Open the advisor tab
9. Observe that there is remediate button

# Before the change


# After the change


# Dependent work link


# Checklist:
